docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -0,0 +1,368 @@
+    "\n",
+    "    return images, labels"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next step is to instantiate pipelines and build them. Building creates and initializes pipeline internals."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-07-28T07:43:41.989183Z",
+     "iopub.status.busy": "2023-07-28T07:43:41.988964Z",
+     "iopub.status.idle": "2023-07-28T07:43:42.104446Z",
+     "shell.execute_reply": "2023-07-28T07:43:42.103668Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Creating pipelines\n",
+      "Building pipelines\n",
+      "<nvidia.dali.pipeline.Pipeline object at 0x7facf81e14b0>\n",
+      "<nvidia.dali.pipeline.Pipeline object at 0x7fad15650bb0>\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Creating pipelines')\n",
+    "training_pipeline = mnist_pipeline(data_path=training_data_path, random_shuffle=True)\n",
+    "validation_pipeline = mnist_pipeline(data_path=validation_data_path, random_shuffle=False)\n",
+    "\n",
+    "print('Building pipelines')\n",
+    "training_pipeline.build()\n",
+    "validation_pipeline.build()\n",
+    "\n",
+    "print(training_pipeline)\n",
+    "print(validation_pipeline)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For DALI pipeline to work with JAX it needs to be wrapped with appropriate DALI iterator. To get the iterator compatible with JAX we need to import it from DALI JAX plugin. In addition to the pipeline we can pass the `output_map`, `reader_name` and `auto_reset` parameters to the iterator. \n",
+    "\n",
+    "**Here is a quick explnation of how these parameters work:**\n",
+    "\n",
+    " - `output_map`: iterators return a dictionary with outputs of the pipeline as its values. Keys in this dictionary are defined by `output_map`. For example, `labels` output returned from the DALI pipeline defined above will be accessible as `iterator_output['labels']`,\n",
+    " - `reader_name`: setting this parameter introduces the notion of an epoch to our iterator. DALI pipeline itself is infinite, it will return the data indefinately, wrapping around the dataset. DALI readers (such as `fn.readers.caffe2` used in this example) have access to the information about the size of the dataset. If we want to pass this information to the iterator we need to point to the operator that should be queried for the dataset size. We do it by naming the operator (note `name=\"mnist_caffe2_reader\"`) and passing the same name as the value for `reader_name` argument,\n",
+    "  - `auto_reset`: this argument controls the behaviour of the iterator after the end of an epoch. If set to `True` will automatically reset the state of the iterator and prepare it to start the next epoch."
```

Comment:
```suggestion
    "  - `auto_reset`: this argument controls the behaviour of the iterator after the end of an epoch. If set to `True`, it will automatically reset the state of the iterator and prepare it to start the next epoch."
```

Reply:
Done